media/starboard/base/endian_util.h

```diff
@@ -0,0 +1,143 @@
+// Copyright 2016 The Cobalt Authors. All Rights Reserved.
```

Comment:
This is also put under `//media/starboard/progressive`.

Reply:
FWIF - @dahlstrom-g IMO it's okay to do "restore" commit and "move" right after, these don't need to be bundled.

Reply:
SGTM.

Reply:
done